seyik/app.js

```diff
@@ -111,6 +110,57 @@ app.post("/posts/register", async (req, res) => {
   res.status(201).json({ message: "postCreated" });
 });
 
+app.patch("/modify", async (req, res) => {
+  const { postingContent, user_id } = req.body;
+
+  await appDataSource.query(
+    `UPDATE posts
+    SET
+    content = ?
+    WHERE posts.user_id = ? 
+    `,
+    [postingContent, user_id]
+  );
+  const rows = await appDataSource.query(
+    `SELECT 
+      users.id as userId,
+      users.name as userName,
+      posts.id as postingId,
+      posts.title as postingTitle,
+      posts.content as postingContent
+      FROM users LEFT JOIN posts
+      ON posts.user_id = users.id
+  `
+  );
+  res.status(200).json({ data: rows });
+});
+
+app.delete("/postsdelete", async (req, res) => {
+  const { postId } = req.body;
+  await appDataSource.query(
+    `DELETE
+    FROM posts
+    WHERE posts.id = ${postId}
+    `
+  );
+  res.status(200).json({ message: "postingDeleted" });
+});
+
+app.post("/like", async (req, res) => {
+  const { userId, postId } = req.body;
+
+  await appDataSource.query(
+    `INSERT INTO likes(
+          user_id,
+          post_id
+      ) VALUES (?, ?);
+      `,
+    [userId, postId]
+  );
+
+  res.status(200).json({ message: "likeCreated" });
```

Comment:
```suggestion
  res.status(201).json({ message: "likeCreated" });
```
status code 수정, 적절한 걸 사용해주세요.